src/idpi/constants.py

```diff
@@ -7,7 +7,13 @@
 
 pc_r_d = 287.05
 pc_r_v = 461.51  # Gas constant for water vapour[J kg-1 K-1]
+pc_rdv = pc_r_d / pc_r_v
+pc_o_rdv = 1.0 - pc_rdv
 pc_cp_d = 1005.0
 pc_rvd = pc_r_v / pc_r_d
 pc_rdocp = pc_r_d / pc_cp_d
 pc_rvd_o = pc_rvd - 1.0
+pc_b1 = 611.21  # Pressure at triple point of water [Pa]
+pc_b2w = 17.502
+pc_b3 = 273.16  # Temperature at triple point of water [K]
+pc_b4w = 32.19  # [K]
```

Comment:
Should we not have descriptions for new constants?

Reply:
thanks, yes. They are constants coming from the literature, I am not sure it is easy to add a description, but I added a reference where they can be found. (The same reference that fieldextra uses)

Reply:
Perfect, thanks